### PR TITLE
fix typos found with CppCheck

### PR DESCRIPTION
**Pull request by AntonioBL**
- [x] I signed [CLA](https://musescore.org/en/cla)
- [x] I made sure the code in the PR follows [the coding rules](https://github.com/musescore/Documentation/blob/master/WorkflowAndGuidelines/CodeGuidelines.md)
- [x] I made sure the code compiles on my machine
- [x] I made sure there are no unnecessary changes in the code
- [x] I made sure the title of the PR reflects the core meaning of the issue you are solving
- [x] I made sure the commit message(s) contain a description and answer the question "Why do those changes fix that particular issue?" or "Why are those changes really necessary as improvements?"
- [N/A] I made sure the commit message title starts with "fix #424242:" if there is a related issue
- [N/A] I created the test (mtest, vtest, script test) to verify the changes I made

### CODE REVIEW

**Comment by Jojo-Schmitz** [comment]
Might be an idea to add that CppCheck to the CI?

**Comment by AntonioBL** [comment]
> 
> 
> Might be an idea to add that CppCheck to the CI?

In principle yes, but at the moment there are literally thousands of warnings (some of them are also false positives). :sweat_smile:

File: mu4/importexport/internal/guitarpro/importgtp-gp6.cpp [hunk 1]
```diff
@@ -1131,10 +1131,6 @@ Fraction GuitarPro6::readBeats(QString beats, GPPartInfo* partInfo, Measure* mea
                                     if (!currentProperty.firstChild().nodeName().compare("Enable")) {
                                         addPalmMute(note);
                                     }
-                                } else if (argument == "Tapped") {
-                                    if (!currentProperty.firstChild().nodeName().compare("Enable")) {
-                                        addTap(note);
-                                    }
```
Comment (igorkorsukov) [inline mu4/importexport/internal/guitarpro/importgtp-gp6.cpp:1137]: why this removed?

Comment (AntonioBL) [reply mu4/importexport/internal/guitarpro/importgtp-gp6.cpp:1137]: Because there is the exact same case at line 1066: it is a duplicated condition.

**Pull request merged**